### ISSUE TITLE
Diverged titles

### ISSUE THREAD

**Issue by Skillmon** [learnlatex/learnlatex.github.io#130]
In some cases the titles of the more details sections have diverged from their lessons. For example, currently the title of lesson-07 is "Including graphics and positioning", while more-07 has the title "More on: Including graphics and making things 'float'". Also the first mark-down formatted heading doesn't match the title in the config block at the top (is that on purpose?).

**Comment by davidcarlisle** [comment]
More specifically lesson-07 is the only lesson-xx.md file that has no top level  `# ` heading and summary, it somehow got missed in the redesign

```
$ grep '^# ' les*md
lesson-01.md:# LaTeX basics
lesson-02.md:# Working with LaTeX
lesson-03.md:# LaTeX document structure
lesson-04.md:# Logical structure
lesson-05.md:# Document classes
lesson-06.md:# Extending LaTeX
lesson-08.md:# Tables
lesson-09.md:# Cross-referencing
lesson-10.md:# Mathematics
lesson-11.md:# Formatting: fonts and spacing
lesson-12.md:# Citations and references
lesson-13.md:# Structuring longer documents
lesson-14.md:# Fonts and Unicode engines
lesson-15.md:# Dealing with errors
lesson-16.md:# Documentation and getting help
```

**Comment by davidcarlisle** [comment]
Addressed with commit  2c301b9 Actually I'd intended to do as a PR @josephwright  can you review?
